R/explain.R

```diff
@@ -290,24 +319,32 @@ explain <- function(model,
                     x_explain,
                     x_train,
                     approach,
+                    paired_shap_sampling = TRUE,
                     prediction_zero,
-                    n_combinations = NULL,
+                    max_n_coalitions = NULL,
+                    adaptive = TRUE,
                     group = NULL,
-                    n_samples = 1e3,
-                    n_batches = NULL,
+                    n_MC_samples = 1e3,
                     seed = 1,
                     keep_samp_for_vS = FALSE,
                     predict_model = NULL,
                     get_model_specs = NULL,
                     MSEv_uniform_comb_weights = TRUE,
-                    timing = TRUE,
                     verbose = 0,
+                    adaptive_arguments = list(),
+                    print_shapleyres = FALSE, # tmp
+                    print_iter_info = FALSE, # tmp
```

Comment:
What's `tmp` here? Probably could be removed

Reply:
the two arguments are now put into verbose instead